pkg/proxy/iptables/proxier.go

```diff
@@ -1164,7 +901,12 @@ func (proxier *Proxier) syncProxyRules() {
 
 	// Build rules for each service.
 	var svcNameString string
-	for svcName, svcInfo := range proxier.serviceMap {
+	for svcName, svc := range proxier.serviceMap {
+		svcInfo, ok := svc.(*serviceInfo)
+		if !ok {
+			glog.Errorf("Failed to cast serviceInfo")
```

Comment:
Would be useful to include svcName in this log message.

Reply:
Make sense, done.